src/config/babelrc.js

```diff
@@ -49,9 +58,13 @@ module.exports = () => ({
         {pragma: isPreact ? 'React.h' : undefined},
       ],
     ),
+    ifAnyDep(['flow-bin'], [optionalRequire('@babel/preset-flow')]),
```

Comment:
This should not be necessary. I think that we should actually include `@babel/preset-flow` in `kcd-scripts`' `dependencies`. (Part of the idea of this package is you generally shouldn't have to install tooling dependencies).

So we can get rid of `optionalRequire`.